.appveyor.yml

```diff
@@ -96,6 +100,15 @@ install:
             (New-Object Net.WebClient).DownloadFile('https://sourceforge.net/projects/wordlist/files/speller/2017.08.24/hunspell-en_GB-ise-2017.08.24.zip', 'c:\spellcheck-dicts\zip\en_GB-ise.zip')
             c:\cygwin\bin\bash -lc "cd /cygdrive/c/spellcheck-dicts/unzip; 7z x -y ../zip/en_US.zip; 7z x -y ../zip/en_GB-ize.zip; 7z x -y ../zip/en_GB-ise.zip"
         }
+    - ps: |
+        if (Test-Path "C:\Program Files (x86)\NSIS\Plugins\StdUtils.dll") {
```

Comment:
its not actually cached anyway is it?

Reply:
it isn

Reply:
well then use the right path at least :P